components/logins/src/logins.udl

```diff
@@ -5,6 +5,9 @@
 namespace logins {
     /// We expose the crypto primitives on the namespace
 
+    /// Initialize NSS
+    void ensure_initialized();
```

Comment:
My one concern with this change is that applications are initializing NSS globally, through one of the components that depends on it.  This seems like small footgun -- if an application stops using `logins`, this could break the `autofill` integration.

How do you feel about adding UniFFI support to `rc_crypto` and making applications call `rc_crypto::ensure_initialized`?

Reply:
Yes, this is also possible, also already (see the comment). I feel like this is a